generator/generators/BaseGenerator.py

```diff
@@ -32,6 +32,8 @@ def render(self, tree):
         env.filters['is_raw_data'] = lambda n: isinstance(n, RawData)
         env.filters['is_archive_resource'] = lambda n: isinstance(n, ArchiveResource)
         env.filters['is_structure'] = lambda n: isinstance(n, Structure)
+        env.filters['is_index'] = lambda n: (isinstance(n, Structure) and
```

Comment:
is_builtin_structure? We might get more than the simple integer based ones

Reply:
In the next line I actually check that this an index.

Reply:
maybe then name it multi_vector_index?

Reply:
Fixed